src/Nethermind/Nethermind.Db/FullPruning/FullPruningDb.cs

```diff
@@ -67,9 +67,9 @@ public FullPruningDb(RocksDbSettings settings, IRocksDbFactory dbFactory, Action
             }
         }
 
-        private void Duplicate(IKeyValueStore db, byte[] key, byte[]? value)
+        private void Duplicate(IKeyValueStore db, ReadOnlySpan<byte> key, byte[]? value)
         {
-            db[key] = value;
+            db[key.ToArray()] = value;
```

Comment:
probably not needed `ToArray`

Reply:
No, the key here is span, and db is indexed by byte array.

Reply:
Actually, now it works... IDE was reindexing.